Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -585,5 +585,68 @@ public class GraphLinter: GraphLinting {
             LintableTarget(platform: .macOS, product: .framework),
             LintableTarget(platform: .macOS, product: .staticFramework),
         ],
+
+        // visionOS products
+        LintableTarget(platform: .visionOS, product: .app): [
+            LintableTarget(platform: .visionOS, product: .staticLibrary),
+            LintableTarget(platform: .visionOS, product: .dynamicLibrary),
+            LintableTarget(platform: .visionOS, product: .framework),
+            LintableTarget(platform: .visionOS, product: .staticFramework),
+            LintableTarget(platform: .visionOS, product: .bundle),
+            LintableTarget(platform: .visionOS, product: .appExtension),
+            LintableTarget(platform: .watchOS, product: .app),
+//            LintableTarget(platform: .watchOS, product: .watchApp),
```

Comment:
I don't believe visionOS apps support embedding watch apps 🤔

![Screenshot 2023-07-06 at 14 42 29](https://github.com/tuist/tuist/assets/11914919/4b0eb42b-640f-413c-8ddf-123145c12f70)

Reply:
Oops, guess I missed that spot... Sorry!